theano/gpuarray/opt.py

```diff
@@ -97,6 +95,15 @@ def f(local_opt):
     return f
 
 
+def register_opt2(tracks, *tags, **kwargs):
```

Comment:
tags and kwargs are just forwarded to the call register() This allow for example to add 'fast_compile' when needed.